script/engine.js

```diff
@@ -681,7 +694,19 @@
 			if(lang && typeof Storage != 'undefined' && localStorage) {
 				localStorage.lang = lang;
 			}
+		},
+
+		setTimeout: function(callback, timeout, skipDouble){
+
+			if( Engine.options.doubleTime && ( typeof skipDouble == "undefined" || skipDouble == false ) ){
+				Engine.log('Double time, cutting timeout in half');
+				timeout = timeout / 2;

```

Comment:
This can be more succinctly written as `timeout /= 2;`
Doesn't really matter to me which way you write it, but I can't help teaching.